### ISSUE TITLE
Number of epochs used for pre-training

### ISSUE THREAD

**Issue by hoangtnm** [facebookresearch/CovidPrognosis#22]
Hi, as I read the paper in section 4.1 it states that `We pretrained for 10 epochs, decaying the learning rate by a factor of 10 each epoch` but the [default configs used for pretraining](https://github.com/facebookresearch/CovidPrognosis/blob/main/cp_examples/moco_pretrain/train_moco.py#L32) uses `200 epochs`, which is different from the paper. Can you confirm which one is the right one?

**Comment by mmuckley** [comment]
Hello @hoangtnm, everything is correct but I can see how it might be confusing. The paper states the pretraining time for **supervised** pretraining as 10 epochs, whereas the config you linked is for **MoCo self-supervised pretraining**. After your comment I don't think we specifically stated the MoCo pretraining time in the paper. I'll look into making a correction.
